pyUltroid/version.py

```diff
@@ -1,2 +1,2 @@
 __version__ = "2023.02.20"
-ultroid_version = "0.8"
+ultroid_version = "0.8.5"
```

Comment:
gg

Reply:
Again this wasn't meant to go over I added .5 to the repo Ive made